src/utils/hub.js

```diff
@@ -215,8 +215,7 @@ function isValidHfModelId(string) {
 export async function getFile(urlOrPath) {
 
     if (env.useFS && !isValidUrl(urlOrPath, ['http:', 'https:', 'blob:'])) {
-        return new FileResponse(urlOrPath.toString());
-
+        return new FileResponse(urlOrPath instanceof URL ? urlOrPath.pathname : urlOrPath);
```

Comment:
Should we perhaps check that the URL uses a valid `file://` schema? Is there any other case where `.pathname` would be valid (on a non-file URL?)

Reply:
update the code

Reply:
Thanks!